lang/cs/Org.Apache.REEF.IO.Tests/TestAzureBlockBlobFileSystemE2E.cs

```diff
@@ -138,6 +139,43 @@ public void TestDeleteE2E()
             Assert.False(CheckBlobExists(blob));
         }
 
+        [Fact(Skip = SkipMessage)]
+        public void TestGetChildrenE2E()
```

Comment:
We should also test listing containers. After all the IFileSystem should work like the local file system. Containers are just root directories from an outside observer.

Reply:
Good idea! I added code to make it work for both containers and blobs.